eth/tracers/api.go

```diff
@@ -799,10 +799,19 @@ func containsTx(block *types.Block, hash common.Hash) bool {
 // TraceTransaction returns the structured logs created during the execution of EVM
 // and returns them as a JSON object.
 func (api *API) TraceTransaction(ctx context.Context, hash common.Hash, config *TraceConfig) (interface{}, error) {
-	_, blockHash, blockNumber, index, err := api.backend.GetTransaction(ctx, hash)
```

Comment:
The pre-existing code seemed a bit odd to me, because the call to `api.backend.GetTransaction()` never returns an error. To make this work, I had to get the `tx` value which was returned and check if its `nil`.

https://github.com/ethereum-optimism/op-geth/blob/c19a58452bf8bde3a8b0656f4a0a1cdeda49e637/eth/api_backend.go#L278-L280